g2p/mappings/utils.py

```diff
@@ -382,7 +392,11 @@ def find_mapping(in_lang: str, out_lang: str) -> list:
         map_in_lang = mapping.get("in_lang", "")
         map_out_lang = mapping.get("out_lang", "")
         if map_in_lang == in_lang and map_out_lang == out_lang:
-            return deepcopy(mapping)
+            if mapping.get("type", "mapping") == "lexicon":
```

Comment:
It's funny to me you bother with a default value when just `mapping.get("type") == "lexicon"` would be equivalent (`None` would also not equal `"lexicon"`).